docs/UserGuide.md

```diff
@@ -112,46 +112,42 @@ If you face any issues while using LinkMeIn, you can refer to the [FAQ Section](
-2. Click on the `LinkMeIn.jar` file to download it.
+2. With Java 11 installed, you can now download our latest jar file from [here](https://github.com/AY2324S1-CS2103T-T17-2/tp/releases/tag/v1.2.0).
 
-![githubRelease.png](images/quick-start/githubRelease.png)
+3. Click on the `LinkMeIn.jar` file to download it.
+    <div style="margin-top: 10px; margin-bottom: 10px">
+        <img src="images/quick-start/githubRelease.png" alt="Github Release" width="650"/>
+    </div>
 
-**Step 3: Choose a home folder for LinkMeIn:**
-1. Decide on a folder to use as the home folder for LinkMeIn. This folder will be used to store the data of LinkMeIn.
-2. Create a new folder (you can name it "LinkMeIn") in a location of your choice, 
-such as your Desktop or Documents folder
-3. Move the downloaded jar file to the folder you have created in the previous step.
+4. Move the downloaded jar file into a home folder of your choice, such as the **Desktop** or **Documents** folder for your LinkMeIn.
 
-**Step 4: Launch LinkMeIn:**
-1. Open up your terminal. If you are unsure of how to do so, you can check out [this FAQ](#2-how-do-i-open-up-my-terminal).
-2. Navigate to the folder where you have placed the jar file with the `cd` command. 
-   - For example if you have placed the jar file in the Documents folder, type `cd Documents` and press Enter.
-3. Type `java -jar LinkMeIn.jar` and press Enter.
+5. Open up your command terminal. If you are unsure of how to do so, you can check out [this FAQ](#2-how-do-i-open-up-my-terminal).
 
-<div markdown="block" class="alert alert-danger">
-**:exclamation: Warning:**<br>
-* Do not move or delete the `data` folder as it contains the data of your applications.
-</div>
+6. Navigate to the folder where you have placed the jar file with the `cd` command. 
+   - For example if you have placed the jar file in the **Documents** folder, type `cd Documents` and press Enter.
+   
+7. Enter the next command `java -jar LinkMeIn.jar` and press Enter. Your command terminal should look like this:
+    <div style="margin-top: 10px; margin-bottom: 10px">
+        <img src="images/quick-start/launchLinkMeIn.png" alt="Launch LinkMeIn" width="650"/>
+    </div>
 
+    <div markdown="block" class="alert alert-danger">
+    **:exclamation: Warning:**<br>
+    * Do not move or delete the `data` folder as it contains the data of your applications.
+    </div>
 
-By now, you should have successfully launched LinkMeIn with the following window appearing on your screen.
+8. You should have successfully launched LinkMeIn with the following interface appearing on your screen.
+    <div style="margin-top: 10px; margin-bottom: 10px">
+        <img src="images/quick-start/startupPage.png" alt="Start Up Page LinkMeIn" width="650"/>
+    </div>
 
-![startupPage.png](images/quick-start/startupPage.png)
+9. Great job! You are now ready to learn about the [Command Line Interface](#using-linkmeins-cli) of LinkMeIn!
```

Comment:
Should we have a hyperlink to the GUI tho? Since the startup image shows the GUI. Will it be better to have a hyperlink to both cli and gui. Or will that be too repetitive?

Reply:
I didn't want to put the GUI here, because I feel that it might be better to link with the CLI.